src/EditorFeatures/VisualBasicTest/Diagnostics/AddImport/AddImportTests_NuGet.vb

```diff
@@ -13,47 +13,50 @@ Imports Moq
-                    WithChangedOption(SymbolSearchOptions.SuggestForTypesInReferenceAssemblies, LanguageNames.VisualBasic, True)
-                Return workspace
-            End Function
-
-            Friend Overrides Function CreateDiagnosticProviderAndFixer(workspace As Workspace, parameters As TestParameters) As (DiagnosticAnalyzer, CodeFixProvider)
-                Dim data = DirectCast(parameters.fixProviderData, ProviderData)
-                Return (Nothing, New VisualBasicAddImportCodeFixProvider(data.Item1, data.Item2))
-            End Function
-
-            Protected Overrides Function MassageActions(actions As ImmutableArray(Of CodeAction)) As ImmutableArray(Of CodeAction)
-                Return FlattenActions(actions)
-            End Function
-
-            <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddImport)>
-            Public Async Function TestSearchPackageSingleName() As Task
-                ' Make a loose mock for the installer service.  We don't care what this test
-                ' calls on it.
-                Dim installerServiceMock = New Mock(Of IPackageInstallerService)(MockBehavior.Loose)
-                installerServiceMock.Setup(Function(i) i.IsEnabled(It.IsAny(Of ProjectId))).Returns(True)
-                installerServiceMock.SetupGet(Function(i) i.PackageSources).Returns(NugetPackageSources)
-                installerServiceMock.Setup(Function(s) s.TryInstallPackage(It.IsAny(Of Workspace), It.IsAny(Of DocumentId), It.IsAny(Of String), "NuGetPackage", It.IsAny(Of String), It.IsAny(Of Boolean), It.IsAny(Of CancellationToken))).
-                                     Returns(True)
-
-                Dim packageServiceMock = New Mock(Of ISymbolSearchService)()
-                packageServiceMock.Setup(Function(s) s.FindPackagesWithTypeAsync(NugetOrgSource, "NuGetType", 0, It.IsAny(Of CancellationToken)())).
-                    Returns(CreateSearchResult("NuGetPackage", "NuGetType", CreateNameParts("NuGetNamespace")))
-
-                Await TestInRegularAndScriptAsync(
+        Private Shared ReadOnly NugetPackageSources As ImmutableArray(Of PackageSource) =
+            ImmutableArray.Create(New PackageSource(NugetOrgSource, "http://nuget.org"))
+
+        Protected Overrides Function CreateWorkspaceFromFile(initialMarkup As String, parameters As TestParameters) As TestWorkspace
+            Dim workspace = MyBase.CreateWorkspaceFromFile(initialMarkup, parameters)
+            workspace.Options = workspace.Options.
+                WithChangedOption(SymbolSearchOptions.SuggestForTypesInNuGetPackages, LanguageNames.VisualBasic, True).
+                WithChangedOption(SymbolSearchOptions.SuggestForTypesInReferenceAssemblies, LanguageNames.VisualBasic, True)
+            Return workspace
+        End Function
+
+        Friend Overrides Function CreateDiagnosticProviderAndFixer(workspace As Workspace) As (DiagnosticAnalyzer, CodeFixProvider)
+            Throw New NotImplementedException()
```

Comment:
It's a required method to implement, but the base type only calls it if you don't implement the higher method.  Since we do implement it, this method will never get called. 

(note: this is a bad inheritance system.  could def be cleaned up.  just not in scope for the work i'm doing here).